experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -153,6 +153,7 @@ export class SharedTree
 		HasListeners<CheckoutEvents>;
 	public readonly view: TreeCheckout;
 	public readonly storedSchema: SchemaEditor<InMemoryStoredSchemaRepository>;
+	private schemaIndexLastChangedSeq: number | undefined;
```

Comment:
```suggestion
	/**
	 * The most recent sequenceNumber on a message processed by this `SharedTree`.
	 * Updated before processing an op, such that reading `currentSeq` while processing an op
	 * gives the sequenceNumber of the op currently being processed.
	 * `undefined` if no message has been processed, e.g. for a detached document or document loaded
	 * from summary without any subsequent ops.
	 * @remarks - Most rebasing is built atop a revision system decoupled from message sequence number.
	 * However, this is sometimes necessary to interop with Fluid runtime APIs, e.g. for incremental summarization.
	 */ 
	private currentSeq: number | undefined;
```

With the update, you should also be updating this unconditionally in `processCore`.

Reply:
Might make more sense to put this comment on `CollabWindow` and reference it from here (say that it's state with the same described semantics)

Reply:
updated `CollabWindow` documentation